src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -237,8 +237,15 @@ RestExecuteOnExtensionResponse handleRestExecuteOnExtensionRequest(RestExecuteOn
                 "No handler for " + ExtensionRestPathRegistry.restPathToString(request.getMethod(), request.getUri())
             );
         }
+        // ExtensionRestRequest restRequest = new ExtensionRestRequest(request);
```

Comment:
Can we remove the commented code?

Reply:
should we consider a possibility of passing the request object as is and extracting the relevant details during object instantiation in ExtensionRestRequest. Doesn't seem like a good idea to pass the whole request object, but wanted to confirm the same

Reply:
Here's the issue discussing it.   https://github.com/opensearch-project/OpenSearch/issues/4429